config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
+
+  root 'static_pages#home'
+  
+  get 'static_pages/home'
```

Comment:
Please remove this file. No need to have this since you already set the root to home.